README.md

```diff
@@ -72,102 +72,71 @@ For guidance on developing for PowerToys, please read the [developer docs](/doc/
-### General
+### Awake
 
-- Reduced resource consumption caused by logging. A thread for each logger was being created even for disabled utilities.
-- The .NET 7 dependency is now shipped self-contained within the utilities, using deep links to reduce storage space usage.
-
-### Color Picker
-
-- Fixed an issue where the custom color formats were not working when picking colors without using the editor.
-- Fixed a crash when using duplicated names for color formats.
-- Added two decimal formats, to distinguish between RGB and BGR.
-- Fixed color name localization, which was not working correctly on 0.65.
+- Disable instead of hiding "Keep screen on" option.
 
 ### FancyZones
 
-- Fixed an editor crash caused by deleting a zone while trying to move it.
-- Reduce the time it takes the tooltip for layout shortcut setting to appear in the editor.
+- Refactored and improved code quality.
 
-### File Locksmith
+### File explorer add-ons
 
-- Fixed an issue causing File Locksmith to hang when looking for open handles in some machines.
+- Fixed escaping HTML-sensitive characters when previewing developer files. Thanks [@davidegiacometti](https://github.com/davidegiacometti)!
 
-### Hosts File Editor
+### Image Resizer
 
-- Added a warning when duplicated entries are detected. Thanks [@davidegiacometti](https://github.com/davidegiacometti)!
+- Improved code quality around a silent crash that was being reported to Microsoft servers.
 
 ### PowerToys Run
 
-- Support drag and dropping for file results. Thanks [@daniel-richter](https://github.com/daniel-richter)!
+- Add option to tab through results only. Thanks [@maws6502](https://github.com/maws6502)!
+- System plugin - Add Recycle Bin command. Thanks [@htcfreek](https://github.com/htcfreek)!
+- WindowWalker plugin - Show all open windows with action keyword. Thanks [@davidegiacometti](https://github.com/davidegiacometti)!
 
 ### Quick Accent
 
-- Added support for dark theme. Thanks [@niels9001](https://github.com/niels9001)!
-- Increased default input delay to improve out of the box experience.
-- Fixed a bug causing the first character to not be selected when opening the overlay. Thanks [@davidegiacometti](https://github.com/davidegiacometti)!
-- Fixed the positioning of the overlay when showing near the horizontal edges of the screen.
-- Added additional Pinyin characters. Thanks [@char-46](https://github.com/char-46)!
-- Added Macedonian characters. Thanks [@ad-mca-mk](https://github.com/ad-mca-mk)!
-- Added a setting to sort characters by usage frequency.
-- Added a setting to always start selection in the first character, even when using the arrow keys as the activation method.
+- Added dashes characters. Thanks [@Aaron-Junker](https://github.com/Aaron-Junker)!
+- Added Estonian characters. Thanks [@jovark](https://github.com/jovark)!
+- Added Hebrew characters. Thanks [@Evyatar-E](https://github.com/Evyatar-E)!
+- Added combining characters. Thanks [@Aaron-Junker](https://github.com/Aaron-Junker)!
```

Comment:
What are combining characters?

Reply:
I believe the word is _diacritic (mark)s_ ?

Reply:
fixed. thanks